App.js

```diff
@@ -7,30 +7,51 @@ import { StyleSheet, Text, View } from 'react-native';
 
 export default class App extends Component {
   constructor() {
-    super()
+    super();
     this.state = {
-      currentPage: 'Login'
+      currentPage: 'Home',
+      currentLatitude: null,
+      currentLongitude: null
     }
   }
 
+  componentDidMount() {
+    this.watchID = navigator.geolocation.watchPosition(
```

Comment:
watchID is a global variable that the watchPosition method will keep updated one app starts to keep track of users' location